stactools_landsat/tests/test_landsat.py

```diff
@@ -0,0 +1,63 @@
+import json
+from pathlib import Path
+
+import pytest
+from pystac import Item
+from stactools.landsat.utils import (
+    stac_api_to_stac,
+    transform_mtl_to_item,
+    transform_stac_to_stac,
+)
+
+TEST_DATA_FOLDER: Path = Path(__file__).parent.joinpath("data")
+LANDSAT_MTL = TEST_DATA_FOLDER / "LC08_L2SR_081119_20200101_20200823_02_T2_MTL.json"
+LANDSAT_STAC = TEST_DATA_FOLDER / "LC08_L2SR_081119_20200101_20200823_02_T2.json"
+
+
+def test_load_json(landsat_mtl, landsat_stac):
+    """Super lazy simple check that we are loading JSON"""
+    json.dumps(landsat_mtl, indent=2)
+    json.dumps(landsat_stac, indent=2)
+
+
+@pytest.mark.skip("Not finished implementing")
+def test_transform_mtl(landsat_mtl):
+    result = transform_mtl_to_item(landsat_mtl)
+
+    # Compare to expected result...
+    print(json.dumps(result, indent=2))
+
+    raise Exception
+
+
+def test_transform_static_stac(landsat_stac):
+    result = transform_stac_to_stac(Item.from_dict(landsat_stac))
+
+    print(json.dumps(result, indent=2))
+
+    item = Item.from_dict(result)
+    item.validate()
+
+
+def test_transform_dynamic_stac(landsat_stac_api_one):
+    result = stac_api_to_stac(landsat_stac_api_one)
+
+    item = Item.from_dict(result)
+    item.validate()
+
+
+@pytest.fixture
```

Comment:
Heads up for the post-Draft PR, we use `unittest` and not `pytest` in the CI, so better to structure this like the other unit tests in the project as `unittest.TestCase`s.

Reply:
Ah, I see. I'll go hunting for other examples and re-implement in unittest.